goreleaser.yaml

```diff
@@ -0,0 +1,11 @@
+builds:
```

Comment:
Might make sense to align this with the scaffold goreleaser config from https://github.com/hashicorp/terraform-provider-scaffolding/blob/main/.goreleaser.yml.

Reply:
Use the scaffold releaser config.